src/test/java/org/gridsuite/filter/server/expertrule/NumberExpertRuleTest.java

```diff
@@ -442,35 +446,58 @@ private static Stream<Arguments> provideArgumentsForLoadTest() {
                 // VoltageLevel fields
                 Arguments.of(EQUALS, FieldType.NOMINAL_VOLTAGE, 13.0, null, load, true),
                 Arguments.of(EQUALS, FieldType.NOMINAL_VOLTAGE, 14.0, null, load, false),
+                // Load fields
+                Arguments.of(EQUALS, FieldType.P0, 77.0, null, load, true),
+                Arguments.of(EQUALS, FieldType.Q0, 277.0, null, load, true),
```

Comment:
Could you change the order as following
Arguments.of(EQUALS, FieldType.P0, 50.0, null, load, false),
Arguments.of(EQUALS, FieldType.Q0, 277.0, null, load, true),

We do the test field by field => easier to manage test data